tools/depends/target/libgpg-error/generate-posix-lock-obj.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+PLATFORM=$1
+
+$PLATFORM/src/gen-posix-lock-obj > $PLATFORM/src/syscfg/tmp.h
+mv $PLATFORM/src/syscfg/tmp.h $PLATFORM/src/syscfg/$(cat $PLATFORM/src/syscfg/tmp.h | head -1 | awk -F '## ' '{print $2}')
```

Comment:
you can do that directly in the Makefile, would be a bit more readable to have it all in one place.
cosmetics:
mv $PLATFORM/src/syscfg/tmp.h $PLATFORM/src/syscfg/$($(AWK) 'NR==1 { print $2 }' $PLATFORM/src/syscfg/tmp.h)

if you prefer your version, please use $(AWK) anyway and put it in the Makefile